File: Sample/app/src/main/java/com/phunware/core/sample/CoreDemo.java
package com.phunware.core.sample;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

public class CoreDemo extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.next).setOnClickListener(v -> {
            Intent i = new Intent(CoreDemo.this, SecondActivity.class);
            startActivity(i);
        });
    }
}

File: Sample/app/src/main/java/com/phunware/core/sample/CoreDemo.kt
package com.phunware.core.sample

import android.app.Activity
import android.os.Bundle
import android.content.Intent
import android.view.View

internal class CoreDemo : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.next).setOnClickListener {
            val intent = Intent(this, SecondActivity::class.java)
            startActivity(intent)
        }
    }
}